.ci/scripts/rm-docker.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+IMAGE="docker.elastic.co/infra/release-manager:latest"
+WORKFLOW="snapshot"
+VERSION="8.2.0"
+BRANCH="main"
+
+# Allow other users write access to create checksum files
+chmod -R 777 build/distributions 
+
+# The "branch" here selects which "$BRANCH.gradle" file of release manager is used
+# VERSION=$(make get-version)
+# MAJOR=$(echo $VERSION | awk -F. '{ print $1 }')
+# MINOR=$(echo $VERSION | awk -F. '{ print $2 }')
+# if [ -n "$(git ls-remote --heads origin $MAJOR.$MINOR)" ] ; then
+#     BRANCH=$MAJOR.$MINOR
+# elif [ -n "$(git ls-remote --heads origin $MAJOR.x)" ] ; then
+#     BRANCH=$MAJOR.x
+# else
+#     BRANCH=main
+# fi
```

Comment:
consider removing this part

Reply:
It will be used soon, it passes the branch param to the release manager docker - Since our branches won't  map out 1to1 to the gradle config files they use per project/branch.